src/python/pants/engine/build_files.py

```diff
@@ -46,7 +46,8 @@ def parse_address_family(address_mapper, directory):
   path_globs = PathGlobs.create('',
                                 include=patterns,
                                 exclude=address_mapper.build_ignore_patterns)
-  files_content = yield Get(FilesContent, PathGlobs, path_globs)
+  snapshot = yield Get(Snapshot, PathGlobs, path_globs)
```

Comment:
In theory we could install a rule in `create_fs_rules` to convert `Snapshot`s to `DirectoryDigest`s, but it doesn't seem like a priority given that I'm probably going to want to make `DirectoryDigest` the "default" API for filesystem access in a followup.

Reply:
Yeah; my instinct was tot not to do this for a couple of reasons:
1. We probably want to change the default
2. Because we're throwing away ordering, having the caller be explicit about this seems potentially useful.

Can change in the future if we decide to :)